salt/metalk8s/addons/dex/deployed/namespace.sls

```diff
@@ -0,0 +1,10 @@
+#!jinja | metalk8s_kubernetes kubeconfig=/etc/kubernetes/admin.conf&context=kubernetes-admin@kubernetes
```

Comment:
As it's directly tied to Dex and its deployment, could be squashed with the chart commit.